source/tanya/math/random.d

```diff
@@ -202,10 +375,14 @@ class Entropy
     {
         allocator.resize(sources, maxSources);
 
-        version (linux)
+        static if (is(PlatformEntropySource))
         {
             this ~= allocator.make!PlatformEntropySource;
         }
+        else version (linux)
```

Comment:
`else`-block can be removed. `linux` always defines an entropy source, so if something fails it would fail above.